src/test/java/plugins/CompressArtifactsPluginTest.java

```diff
@@ -89,9 +91,25 @@ public void archiveLargerThan4GInTotal() throws Exception {
         FreeStyleJob job = jenkins.jobs.create();
         job.configure();
         job.addPublisher(ArtifactArchiver.class).includes("*");
-        job.addBuildStep(ShellBuildStep.class).command( // Generate archive larger than 4G
+        if (SystemUtils.IS_OS_UNIX) {
+            job.addBuildStep(ShellBuildStep.class).command( // Generate archive larger than 4G
                 "#!/bin/bash\nwget $JENKINS_URL/jnlpJars/jenkins-cli.jar -O stuff.jar; for i in {0..7000}; do cp -l stuff.jar stuff.${i}.jar; done"
-        );
+                                            );
+        }
+        else {
+            // On windows although we could create hard links this would exceed the number of hard links you can create for a given file. 
+            // and sparse files compress too well. 
+            //job.addBuildStep(BatchCommandBuildStep.class).command("for /l %%x in (0, 1, 7000) do fsutil file createnew stuff.%%x.jar 819200");
+            job.addBuildStep(BatchCommandBuildStep.class).command("powershell -command \"& { " +
+                                            "try { " +
```

Comment:
I guess this should have a similar `assumeWgetOrPowerShellv3` thingie as you've done in other places?
But since the test didn't have that before I don't see it as any blocker, just a nice to have.